NEWS.md

```diff
@@ -3,6 +3,10 @@ there might not be any notable changes in new versions of this project.
 
 # NEWS
 
+## 6.4.5 (December 6, 2024)
```

Comment:
```suggestion
```

We're going to add the version number and release date when this is released. In the meantime, we just add to the top as unreleased.